ios/Classes/PTFlutterViewController.h

```diff
@@ -14,6 +14,24 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL needsRemoteDocumentLoaded;
 @property (nonatomic) BOOL documentLoaded;
 
+// viewer options
+@property (nonatomic, assign) BOOL showNavButton;
+
+@property (nonatomic, assign, getter=isReadOnly) BOOL readOnly;
+@property (nonatomic, getter=isThumbnailEditingEnabled) BOOL thumbnailEditingEnabled;
+@property (nonatomic, getter=isContinuousAnnotationEditing) BOOL continuousAnnotationEditing;
+
+@property (nonatomic, copy, nullable, getter=getAnnotationAuthor) NSString* annotationAuthor;
```

Comment:
Don't rename getters with "get". Just use the default. Prefixing get goes against obj-c conventions.